README.md

```diff
@@ -0,0 +1,52 @@
+# React BookStore
+
+> "React BookStore" is a project that displays a list of books and allows you to add and remove books from that list.
+
+# Built With
+
+- CSS
+- React JS
+
+# Screenshot
+
+![screenshot](https://github.com/olawale-o/react-bookstore/blob/add-styling/src/assets/screenshot.png?raw=true")
+
+# Getting started
+
+# Install
+
+```bash
+npm install
+```
+
+# Usage
+
+```bash
+npm start
+```
+
+# Run tests
+
+```bash
+npm test
+```
+
+**Yahya EL Ganayni**
+
+- GitHub: [@yahyaELganayni](https://github.com/yahyaelganyni1)
+- Twitter: [@yahyaELganayni](https://twitter.com/@crazy20046)
+- LinkedIn: [yahyaELganayni](https://www.linkedin.com/in/yahya-el-ganayni-a456115b/)
+
+# Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page.](https://github.com/olawale-o/react-bookstore/issues)
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## License
+
+- This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] You might want to think about fixing the issues page link because you are using one from another person.
- [ ]  Maybe you could add the MIT file because it is missing.
- These suggestions will help you to have a professional Readme file. 🔖